src/vuex/modules/kuzzle/store.ts

```diff
@@ -202,8 +202,12 @@ const actions = createActions({
 
     loadedEnv = JSON.parse(localStorage.getItem(LS_ENVIRONMENTS) || '{}')
     Object.keys(loadedEnv).forEach(envName => {
+      const env = loadedEnv[envName]
+      if (env.hideAdminBanner === undefined) {
+        env.hideAdminBanner = ['localhost', '127.0.0.1'].includes(env.host)
```

Comment:
This array should be factorized into a dedicated constant (or even be determined from the conf), to avoid annoying duplication.